music_assistant/providers/tidal/__init__.py

```diff
@@ -495,6 +496,40 @@ async def _post_data(
                     await self._handle_response(response, return_etag=False),
                 )
 
+    @prepare_api_request
+    async def _put_data(
+        self,
+        endpoint: str,
+        data: dict[str, Any] | None = None,
+        as_form: bool = False,
+        **kwargs: Any,
+    ) -> dict[str, Any]:
+        """Send PUT data to Tidal API."""
+        # Use BASE_URL_V2 for PUT requests to mixes endpoints
+        base_url = kwargs.pop(
+            "base_url", self.BASE_URL_V2 if "mixes" in endpoint else self.BASE_URL
+        )
+        url = f"{base_url}/{endpoint}"
+
+        if as_form:
+            # Set content type for form data
+            headers = kwargs.get("headers", {})
+            headers["Content-Type"] = "application/x-www-form-urlencoded"
+            kwargs["headers"] = headers
+            # Use data parameter for form-encoded data
+            async with self.mass.http_session.put(url, data=data, **kwargs) as response:
+                return cast(
+                    "dict[str, Any]",
+                    await self._handle_response(response, return_etag=False),
+                )
+        else:
+            # Use json parameter for JSON data (default)
+            async with self.mass.http_session.put(url, json=data, **kwargs) as response:
+                return cast(
+                    "dict[str, Any]",
+                    await self._handle_response(response, return_etag=False),
+                )
```

Comment:
```suggestion
        # Use json parameter for JSON data (default)
        async with self.mass.http_session.put(url, json=data, **kwargs) as response:
            return cast(
                "dict[str, Any]",
                await self._handle_response(response, return_etag=False),
            )
```
There is a return above so no need for the else